docker-compose.yml

```diff
@@ -0,0 +1,106 @@
+version: '3.8'
+
+services:
+  nginx:
+    image: ghcr.io/ai-cfia/nginx:main
+    container_name: fertiscan-nginx
+    restart: always
+    ports:
+      - "80:80"
+    environment:
+      - BACKEND_PORT=5000
+      - FRONTEND_PORT=3000
+    depends_on:
+      - frontend
+      - backend
+    networks:
+      - fertiscan-network
+
+  frontend:
+    image: ghcr.io/ai-cfia/fertiscan-frontend:main
+    container_name: frontend
+    restart: always
+    ports:
+      - "3000:3000"
+    env_file:
+      - .env
```

Comment:
Is it fine that we use the same env file for the services? I don't have an opinion on this.

Reply:
Yes. Variables from the frontend are different so they wont interfere. If needed the user can create a different file and update this line.